bundles/org.openhab.binding.yamahamusiccast/src/main/java/org/openhab/binding/yamahamusiccast/internal/YamahaMusiccastHandler.java

```diff
@@ -590,6 +590,16 @@ private void updateStateFromUDPEvent(String zoneToUpdate, UdpMessage targetObjec
                 break;
         }
 
+        logger.debug("{} - Response: {}", zoneToUpdate, responseCode);
+        logger.debug("{} - Power: {}", zoneToUpdate, powerState);
+        logger.debug("{} - Mute: {}", zoneToUpdate, muteState);
+        logger.debug("{} - Volume: {}", zoneToUpdate, volumeState);
+        logger.debug("{} - Volume in dB: {}", zoneToUpdate, (actualVolume != null) ? actualVolume.getValue() : "");
+        logger.debug("{} - Max Volume: {}", zoneToUpdate, maxVolumeState);
+        logger.debug("{} - Input: {}", zoneToUpdate, inputState);
+        logger.debug("{} - Soundprogram: {}", zoneToUpdate, soundProgramState);
+        logger.debug("{} - Sleep: {}", zoneToUpdate, sleepState);
```

Comment:
Trace level would be more appropriate here. Can be surrounded by `if (logger.isTraceEnabled())` for performance.